pkg/cmd/tknpac/resolve/resolve.go

```diff
@@ -71,6 +71,11 @@ func Command(run *params.Run) *cobra.Command {
 					return err
 				}
 			}
+
+			if err := run.GetConfigFromConfigMap(ctx); err != nil {
+				log.Printf("Warning: cannot get pipelines-as-code config map in pipelines-as-code namespace. Using defaults. Error: %v", err)
+			}
```

Comment:
does log.printf adds a \n at the end ? i can't rem..

and isn't that %w for wrapping the error?

Reply:
It does not, you would need a `\n`. `%v` is to display "complex" struct.

Reply:
updated!